constants.py

```diff
@@ -3,9 +3,9 @@
 """Signals and constants used in the application."""
 
 MUTE_CHECKBOX = 'mute_checkbox'
-MUTE_ALL_STREAMS = 'mute_all_streams'
-ADD_NEW_STREAM = 'add_new_stream'
-EXPORT_STREAMS_TO_CLIPBOARD = 'export_streams_to_clipboard'
+MUTE_ALL_STREAMS = 'MuteAllStreams'
```

Comment:
These are not underscored because of 💣 The reason is described [here](https://github.com/kaszim/desktop-stream-viewer/pull/81#discussion_r115099100)

But we can change to some other styling if that floats you boat.